lib/src/generator/templates.dart

```diff
@@ -1,9 +1,7 @@
 import '../utils/utils.dart';
 import 'label.dart';
 
-String generateL10nDartFileContent(
-    String className, List<Label> labels, List<String> locales,
-    [bool otaEnabled = false]) {
+String generateL10nDartFileContent(String className, List<Label> labels, List<String> locales, [bool otaEnabled = false]) {
```

Comment:
Can we reformat this line with Dart formatter (e.g. run _"Reformat Code with dartfmt"_ context action in Android Studio) as `pub.dev` site will run a static analysis of the code and it will probably complain?